src/VisualStudio/Core/Def/Implementation/Diagnostics/VisualStudioDiagnosticAnalyzerExecutor.cs

```diff
@@ -151,6 +151,11 @@ private CompilationWithAnalyzers CreateAnalyzerDriver(CompilationWithAnalyzers a
 
         private void ReportAnalyzerExceptions(Project project, ImmutableDictionary<DiagnosticAnalyzer, ImmutableArray<DiagnosticData>> exceptions)
         {
+            if (exceptions == null)
```

Comment:
that is breaking change not suitable for RTW. this change is safe change that doesnt require LUT and Razor to update their bit for RTW as well.

and that kind of change, sure we can change after RTW. with that reasoning, I didn't update any caller which is not officially on in RTW such as FAR, GOTO and etc.